dts/x86/intel_curie.dtsi

```diff
@@ -166,7 +166,7 @@
 				reg = <0x0>;
 				label = "cc2520";
 				spi-max-frequency = <8000000>;
-				status = "disabled";
+				status = "ok";
```

Comment:
Why this change here?  Typically done in the board.dts

Reply:
that's due to a wrong rebase, move it to proper commit

Reply:
? shouldn't the addition of the cc2520@0 node & cs-gpios be in the board.dts.  Might still be a rebase issue.

Reply:
no you'r right, cc2520 is present on the board, not on the soc